misc/package/Data Files/MWSE/core/mcm/main.lua

```diff
@@ -295,10 +314,120 @@ end
 
 local function cleanupMCM(e)
 	currentModConfig = nil
+	---@diagnostic disable-next-line: cast-local-type
 	modConfigContainer = nil
 	previousModConfigSelector = nil
 end
 
+
+local function initializeKeybindTemplate()
+	local KEYBINDER_CLASSES = {
+		KeyBinder = true,
+		MouseBinder = true,
+	}
+	---@param keybindMenuCategory mwseMCMCategory
+	---@param modTemplate mwseMCMTemplate
+	---@param modCategory mwseMCMCategory
+	local function addAllKeybinds(keybindMenuCategory, modTemplate, modCategory)
```

Comment:
Looks like this function doesn't use any upvalues. In such cases, I prefer to define such functions outside other functions.